presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveCoercion.java

```diff
@@ -106,6 +110,11 @@
                         "    int_to_bigint              INT," +
                         "    bigint_to_varchar          BIGINT," +
                         "    float_to_double            " + floatToDoubleType + "," +
+                        "    double_to_float            DOUBLE," +
+                        "    shortdecimal_to_shortdecimal          DECIMAL(10,2)," +
+                        "    shortdecimal_to_longdecimal          DECIMAL(10,2)," +
+                        "    longdecimal_to_shortdecimal          DECIMAL(20,12)," +
+                        "    longdecimal_to_longdecimal          DECIMAL(20,12)," +
```

Comment:
What about `float to decimal`, `double to decimal`, `decimal to float`, `decimal to double`?

Reply:
Will you implement tests for those conversions too?

Reply:
@sopel39 Yes, I am working on it. Already labelled WIP. Once finishing, will remove WIP.

Reply:
I am fixing all your previous comments.